app/views/layouts/_header.html.erb

```diff
@@ -10,25 +10,25 @@
   <div class="collapse navbar-collapse" id="collapsedNavbar">
     <ul class="navbar-nav ml-auto mt-2 mt-lg-0">
       <li class="nav-item px-2">
-        <button class="btn btn-outline-success my-2 my-sm-0 rounded-0 simulator text-dark" type="submit">Simulator</button>
+        <a class="btn btn-outline-success my-2 my-sm-0 rounded-0 simulator text-dark" href="/simulator">Simulator</button>
       </li>
-      <li class="nav-item px-2 active">
```

Comment:
Can we also make sure that active class applied properly based on the current page

Reply:
This should be done

Reply:
I guess @tachyons meant to add the active class to an element only when that particular page is active; rather than removing it altogether.